xhydro/cc.py

```diff
@@ -0,0 +1,27 @@
+"""Module to compute climate change statistics using xscen functions."""
+
+# Special imports from xscen
+from xscen import (
+    climatological_mean,  # FIXME: To be replaced with climatological_op once available
+)
+from xscen import compute_deltas, ensemble_stats, produce_horizon
+
+__all__ = [
+    "climatological_op",
+    "compute_deltas",
+    "produce_horizon",
+    "ensemble_stats",
+]
+
+
+# FIXME: To be deleted once climatological_op is available in xscen
+def climatological_op(ds, **kwargs):
+    """Compute climatological operation.
+
+    Notes
+    -----
+    This is a temporary wrapper to be deleted once climatological_op is available in xscen.
+    For the time being, it is a simple wrapper around climatological_mean.
+    See :py:func:`xscen.climatological_mean` for more details.
```

Comment:
@Zeitsperre This does not seem to link to the xscen documentation once on RtD... Any idea why?

Reply:
`xscen` is not within the scope of the project. This is a case for intersphinx, so it needs to be called differently.

Reply:
hmm oh I see. This is strange.

Reply:
Looks like we can't just shortcut the link! This should be fixed now.